source/connect/required-access.txt

```diff
@@ -0,0 +1,97 @@
+.. _required-access:
+
+===============
+Required Access
+===============
+
+.. default-domain:: mongodb
+
+With authentication/authorization enforced on the connected MongoDB
+instance, the user requires appropriate privileges to access various
+features. The following table lists the privileges required to access
+the features as well as the built-in roles that can provide these
+privileges:
+
+.. note::
```

Comment:
I think we should move this to the bottom of the table. It currently interrupts the flow of the second sentence on this page.